code/src/SubmitButton.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const SubmitButton = () => {
+  return (
+    <button className="submit-button" type="submit">
+      Submit my answers
+    </button>
+  )
```

Comment:
To make this component more usable you could have sent props and then used these props to set a different class, different button type, different text etc, and then re-used it for each question.